lib/util/insightspublisher/index.js

```diff
@@ -60,6 +62,9 @@ class InsightsPublisher extends EventEmitter {
       },
       _WebSocket: {
         value: options.WebSocket
+      },
+      _isIpad: {
```

Comment:
Why create another internal property? Can we call util.isIpad() directly on line 205?